pyfar/classes/audio.py

```diff
@@ -847,6 +967,30 @@ def __iter__(self):
         """
         return _SignalIterator(self._data.__iter__(), self)
 
+    def _check_conjugate_symmetry(self):
+        """Check if the frequency bins are conjugate symmetric
+        around 0 Hz.
+
+        Returns
+        -------
+        results : bool
+            return `True` if frequency data are conjugate symmetric around
+            0 Hz, return `False` if not.
+
+        """
+        idx = self.find_nearest_frequency(0)
+
+        if fft._is_odd(self._n_samples):
+            mirror_spec = np.conj(np.flip(self._data[..., :idx], axis=-1))
+        else:
+            mirror_spec = np.conj(np.flip(self._data[..., 1:idx], axis=-1))
+
+        if mirror_spec.shape[-1] > 0 and np.allclose(
+                self._data[..., idx+1:], mirror_spec, atol=1e-15):
```

Comment:
Maybe make the tolerance depend on the dtype of the data:
```suggestion
                self._data[..., idx+1:], mirror_spec, atol=5*np.finfo(self._data.dtype).eps:
```